e2e/rook-test-framework/rook-infra-manager/scripts/rook-dind-cluster-v1.6.sh

```diff
@@ -0,0 +1,1045 @@
+#!/bin/bash
+# Copyright 2017 Mirantis
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
i would just do something like:
> The following modifications have been made to the code in this file:

And then just start a bullet list that succinctly summarizes the changes you made.  someone else can chime in if there is a specific format we need to use as per licensing, but I don't know of one.  The intent is that you list the modifications you've made.